### PR TITLE
Updated Happy Base framework to the latest version of Bigtable 0.29.0

### PR DESCRIPTION
**Pull request by vikas-jamdar**
setup.py upadated

line 54:  'google-cloud-bigtable >= 0.29.0',
line 61: version='0.29.0',

### CODE REVIEW

**Comment by vikas-jamdar** [comment]
I signed it!

**Comment by vikas-jamdar** [comment]
I signed it!

**Comment by tseaver** [comment]
Thanks for the patch!

File: setup.py [hunk 1]
```diff
@@ -51,14 +51,14 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-bigtable >= 0.26.0, < 0.27dev',
+    'google-cloud-bigtable >= 0.29.0',
```
Comment (tseaver) [inline setup.py]: Please leave an upper bound in place, e.g.:

```python
REQUIREMENTS = [
    'google-cloud-bigtable >= 0.29.0, < 0.30dev',
]
```

**Comment by tseaver** [comment]
We also need to deal with [test breakage](https://travis-ci.org/GoogleCloudPlatform/google-cloud-python-happybase/builds/386360152) due to a [breaking change](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4745) in `google-cloud-bigtable`:  the `timestamp` argument to `row_data.Cell` is now named `timestamp_micros`, and is no longer a `datetime.datetime`, but just the integer microseconds since epoch.

**Comment by tseaver** [comment]
Thanks very much for your efforts!

**Pull request merged**